src/nodes.coffee

```diff
@@ -1748,6 +1748,13 @@ exports.Assign = class Assign extends Base
     vvarText = fragmentsToText vvar
     assigns  = []
     expandedIdx = false
+    # check if variable is object and containes splat, e.g. {a, b, c...}
+    hasSplat = isObject and (i for obj, i in objects when obj instanceof Splat).length == 1
```

Comment:
No. We need to show a nice error message, like when you try to use multiple splats in array desctructuring or parameter list.